tox.ini

```diff
@@ -39,6 +39,7 @@ allowlist_externals =
 [testenv:docs]
 extras =
     docs
+    extras
```

Comment:
is this common practice? 
extras = 
  extras
  
 ?

Reply:
I named the extra libraries `extras` as I couldn't come up with anything more spectacular. If you have a better name in mind, go for it

Reply:
not really haha, sorry for complaining without a propose solution, let's keep it like this